docs/LV_client/tutorials/Simulator_configuration.md

```diff
@@ -0,0 +1,9 @@
+## Simulator Configuration
+
+This tutorial demonstrates how to use the LabVIEW client to configure the behavior of vehicles in the scene as well as the object's materials when using the monoDrive Simulator. 
+C++, Python, ROS, Simulink, and Matlab client are also available.  
```

Comment:
I think it's alright to eliminate the "also here: C++, Python, ROS, Simulink, and Matlab client are available.